lib/src/Uncertainty/Bayesian/openturns/RandomWalkMetropolisHastings.hxx

```diff
@@ -98,6 +98,7 @@ protected:
 private:
   // proposal densities of the markov chain
   Distribution proposal_;
+  Bool proposalIsSymmetric_;
```

Comment:
```Bool proposalIsSymmetric_ = false;``` so that its always initialized, even in the default ctor

btw it should rather be named "isProposalSymmetric_" for consistency with what can be found in other classes

Reply:
Done